gremlin-groovy/src/main/java/org/apache/tinkerpop/gremlin/groovy/jsr223/GremlinGroovyScriptTestEngine.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.tinkerpop.gremlin.groovy.jsr223;
+
+import org.apache.tinkerpop.gremlin.jsr223.GremlinScriptEngineFactory;
+import org.apache.tinkerpop.gremlin.language.translator.GremlinTranslator;
+import org.apache.tinkerpop.gremlin.language.translator.Translator;
+
+import javax.script.ScriptContext;
+import javax.script.ScriptException;
+
+/**
+ * Designed to be used in tests.
+ * Converts incoming script from gremlin-lang to gremlin-groovy compatible form and executes with
+ * {@code GremlinGroovyScriptEngine}.
+ */
+
+public class GremlinGroovyScriptTestEngine extends GremlinGroovyScriptEngine {
```

Comment:
This class is for tests but only by convention because it is in main `gremlin-groovy` code. Can it be actually moved to test code somehow so that by design it is not usable any other way?

Reply:
moved to gremlin-server tests